docs/reference/filter-reference.md

```diff
@@ -35,9 +35,7 @@ Currently, the Ambassador Edge Stack supports four filter types: `External`, `JW
 
 ### Filter Type: `External`
 
-The `External` filter type exposes the Ambassador Edge Stack `AuthService` interface to external authentication services. This is useful in a number of situations, e.g., if you have already written a custom `AuthService`, but also want to use other filters.
-
-The `External` filter looks very similar to an `AuthService` annotation:
+The `External` filter calls out to an external service speaking the [`ext_authz` protocol][], providing a highly flexible interface to plug in your own authentication, authorization, and filtering logic.
```

Comment:
What does `[ext_authz protocol][]` mean in Markdown? :joy:

Reply:
@LukeShu did you mean to reference something here?